README.md

```diff
@@ -75,22 +60,42 @@ In Go land:
 
 ### Node.js
 
-Install `interface-ipfs-core` as one of the dependencies of your project and as a test file. Then, using `mocha` (for Node.js) or a test runner with compatible API, do:
+Install `interface-ipfs-core` as one of the dependencies of your project. Install the following test file:
 
-```
-var test = require('interface-ipfs-core')
+```js
+/* eslint-env mocha */
+'use strict'
+
+const test = require('interface-ipfs-core')
+const IPFS = require('ipfs')
 
-var common = {
+const common = {
   setup: function (cb) {
-    cb(null, yourIPFSInstance)
+    const ipfs = new IPFS()
+    ipfs.load(() => {
+      cb(null, ipfs)
+    })
   },
   teardown: function (cb) {
     cb()
   }
 }
 
-// use all of the test suits
-test.all(common)
+test.files(common)
```

Comment:
This is only an example of running the `files` tests, not all of them, this should be noted with a reference to where one can find a list of all of them